### DISCUSSION TITLE
Instalation

### DISCUSSION THREAD

**Discussion by pepeballesterostel** [huggingface/diffusers#7774] [Q&A]
For the installation of diffusers in colab, I am following the [colab example](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/diffusers_intro.ipynb#scrollTo=e5MYkuhcRGAS). This just to work just fine, but now, after installing `diffusers==0.11.1`, I cannot import the diffusers library:
`AttributeError: module 'jax.random' has no attribute 'KeyArray'`

JAX version in colab 0.4.26 is compatible with that of diffusers dependencies:  `"jax>=0.4.1", "jaxlib>=0.4.1"`,
   Any help to install correctly diffusers in colab is much appreciated!

**Comment by tolgacangoz** [comment]
Could you just remove `==0.11.1` part, and try with the latest version?
